React 18 conflict dependencies

I can't run `npm install` anymore once I installed react-ogl in a clean Next project due to conflicting dependencies between React 18 and React ^17. What am I missing?

NPM version: 8.18.0
Node version: 16.15.0

The log:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: react-native@0.64.3
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   peer react@"^17.0.2 || ^18.0.0-0" from next@12.2.5
npm ERR!   node_modules/next
npm ERR!     next@"12.2.5" from the root project
npm ERR!   peer react@"^18.2.0" from react-dom@18.2.0
npm ERR!   node_modules/react-dom
npm ERR!     peer react-dom@"^17.0.2 || ^18.0.0-0" from next@12.2.5
npm ERR!     node_modules/next
npm ERR!       next@"12.2.5" from the root project
npm ERR!     peerOptional react-dom@">=17.0" from react-ogl@0.6.4
npm ERR!     node_modules/react-ogl
npm ERR!       react-ogl@"^0.6.4" from the root project
npm ERR!     2 more (react-use-measure, the root project)
npm ERR!   8 more (react-ogl, react-use-measure, styled-jsx, suspend-react, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"17.0.1" from react-native@0.64.3
npm ERR! node_modules/react-native
npm ERR!   peer react-native@">=0.64.0-rc.0 || 0.0.0-*" from @react-native-community/cli@5.0.1
npm ERR!   node_modules/react-native/node_modules/@react-native-community/cli
npm ERR!     @react-native-community/cli@"^5.0.1-alpha.1" from react-native@0.64.3
npm ERR!   peerOptional react-native@">=0.64" from react-ogl@0.6.4
npm ERR!   node_modules/react-ogl
npm ERR!     react-ogl@"^0.6.4" from the root project
npm ERR!   1 more (@expo/browser-polyfill)
npm ERR!
npm ERR! Conflicting peer dependency: react@17.0.1
npm ERR! node_modules/react
npm ERR!   peer react@"17.0.1" from react-native@0.64.3
npm ERR!   node_modules/react-native
npm ERR!     peer react-native@">=0.64.0-rc.0 || 0.0.0-*" from @react-native-community/cli@5.0.1
npm ERR!     node_modules/react-native/node_modules/@react-native-community/cli
npm ERR!       @react-native-community/cli@"^5.0.1-alpha.1" from react-native@0.64.3
npm ERR!     peerOptional react-native@">=0.64" from react-ogl@0.6.4
npm ERR!     node_modules/react-ogl
npm ERR!       react-ogl@"^0.6.4" from the root project
npm ERR!     1 more (@expo/browser-polyfill)
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/arno/.npm/eresolve-report.txt for a full report.
```

package.json
```
{
  "name": "my-app",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "next": "12.2.5",
    "ogl": "^0.0.97",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-ogl": "^0.6.4"
  },
  "devDependencies": {
    "@types/node": "18.7.13",
    "@types/react": "18.0.17",
    "@types/react-dom": "18.0.6",
    "eslint": "8.22.0",
    "eslint-config-next": "12.2.5",
    "typescript": "4.7.4"
  }
}

```

--- a/package.json
+++ b/package.json
@@ -54,7 +54,6 @@
     "vite": "^3.0.9"
   },
   "dependencies": {
-    "@expo/browser-polyfill": "^1.0.0",
     "@types/ogl": "npm:ogl-types@^0.0.97",
     "@types/react-reconciler": "^0.26.7",
     "@types/webxr": "^0.5.0",

--- a/src/Canvas.native.tsx
+++ b/src/Canvas.native.tsx
@@ -5,7 +5,6 @@ import { Block, SetBlock, ErrorBoundary } from './utils'
 import { events as createTouchEvents } from './events'
 import { RenderProps } from './types'
 import { render, unmountComponentAtNode } from './renderer'
-import '@expo/browser-polyfill'
 
 export interface CanvasProps extends Omit<RenderProps, 'size' | 'dpr'>, ViewProps {
   children: React.ReactNode

--- a/tests/utils/setupTests.ts
+++ b/tests/utils/setupTests.ts
@@ -102,7 +102,6 @@ jest.mock('react-native', () => ({
   ),
 }))
 jest.mock('react-native/Libraries/Pressability/Pressability.js', () => ({}))
-jest.mock('@expo/browser-polyfill', () => ({}))
 jest.mock('expo-gl', () => ({
   GLView: ({ onContextCreate }: { onContextCreate: any }) => {
     React.useLayoutEffect(() => {

